aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.csv

```diff
@@ -156,14 +156,14 @@ mission:taxi:duration,0.1677,h
 settings:equations_of_motion,2DOF
 settings:mass_method,GASP
 
-# Initial Guesses
+# Initialization Guesses
 actual_takeoff_mass,0
 climb_range,0
 cruise_mass_final,0
 flight_duration,0
-fuel_burn_per_passenger_mile,0.1
```

Comment:
Is there any ramification to zeroing out these initial conditions? Seems they are in all of the main benchmark-test GASP missions.

Reply:
no, if these variables are unset (or set to 0) they will be replaced with default values when the csv is read in. The values here (.1 and .99) are the default values that they will be replaced with, basically the defaults were specified twice for these two values, so I made them consistent with the rest of the variables here